parser/SqlParser.ypp

```diff
@@ -543,6 +555,15 @@ create_table_statement:
     $$ = new quickstep::ParseStatementCreateTable(@1.first_line, @1.first_column, $3, $5);
   };
 
+create_index_statement:
```

Comment:
Sure Craig, your concerns for supporting indices on pre-existing blocks is absolutely right and something that I also discussed here with Jignesh and Marc. So for now, the way we have decided is to just build indices on the new ones and not worry about the pre-existing blocks.